wdl/AnnotateExternalAF.wdl

```diff
@@ -223,7 +223,9 @@ task SplitVcf{
     
     command <<<
         svtk vcf2bed -i SVTYPE -i SVLEN ~{vcf} tmp.bed
-        cut -f1-4,7-8 tmp.bed > ~{prefix}.bed
+        head -1 tmp.bed > tmp.sorted.bed
+        awk 'NR > 1' < tmp.bed | sort -k1,1V -k2,2n -k3,3n >> tmp.sorted.bed
+        cut -f1-4,7-8 tmp.sorted.bed > ~{prefix}.bed
```

Comment:
I think you can eliminate one of the temp files using `-` to direct `vcf2bed` to stdout:
```
svtk vcf2bed -i SVTYPE -i SVLEN ~{vcf} - \
  | cut -f1-4,7-8 > tmp.bed
head -1 tmp.bed > ~{prefix}.bed
awk 'NR > 1' < tmp.bed \
  | sort -k1,1V -k2,2n -k3,3n >> ~{prefix}.bed
rm tmp.bed
```
Also good to delete the temp file when done